projekte/2015-06-12-be-oparl.html

```diff
@@ -1,21 +1,22 @@
 ---
 layout: project
-lab: OK Lab Berlin
+lab: OK Labs Berlin, München, Ruhrgebiet
```

Comment:
Mit `OK Lab Berlin, OK Lab München, OK Lab Ruhrgebiet` sollte es in den Projektübersichten der drei Labs auftauchen.  Allerdings wird in der Projektbox dann kein Lab mehr angezeigt.

- https://github.com/okfde/codefor.de/blob/c5b128dc47be9813d80f779afda17a1dde7aaa24/_includes/project-teaser.html#L23-L32
- https://github.com/okfde/codefor.de/blob/c5b128dc47be9813d80f779afda17a1dde7aaa24/_includes/project-teaser-slim.html#L34-L43